### ISSUE TITLE
Windows batch files (.bat) are not distributed

### ISSUE THREAD

**Issue by scharlton2** [conda-forge/panoply-feedstock#7]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Batch files aren't installed on windows clients.

Since this project is `noarch`  (it seems that) only the build.sh file is used during the build (no bld.bat).  The only way to run `panoply` on a windows system is to use a shell (such as `git bash`).

### Installed packages

```shell
❯ micromamba list
List of packages in environment: "C:\\Users\\charlton\\micromamba\\envs\\panoply-issue"

  Name                 Version       Build        Channel
---------------------------------------------------------------
  openjdk              22.0.1        h57928b3_1   conda-forge
  panoply              5.5.0         hd8ed1ab_0   conda-forge
  symlink-exe-runtime  1.0           hcfcfb64_0   conda-forge
  ucrt                 10.0.22621.0  h57928b3_0   conda-forge
  vc                   14.3          h8a93ad2_20  conda-forge
  vc14_runtime         14.40.33810   hcc2c482_20  conda-forge
  vs2015_runtime       14.40.33810   h3bf8584_20  conda-forge
```

### Environment info

```shell
❯ micromamba info

       libmamba version : 1.5.3
     micromamba version : 1.5.3
           curl version : libcurl/8.2.1-DEV Schannel zlib/1.2.13
     libarchive version : libarchive 3.6.2 zlib/1.2.13 liblzma/5.4.3 bz2lib/1.0.8 liblz4/1.9.4 libzstd/1.5.5
       envs directories : C:\Users\charlton\micromamba\envs
          package cache : C:\Users\charlton\micromamba\pkgs
                          C:\Users\charlton\.mamba\pkgs
                          C:\Users\charlton\AppData\Roaming\.mamba\pkgs
            environment : panoply-issue (active)
           env location : C:\Users\charlton\micromamba\envs\panoply-issue
      user config files : C:\Users\charlton\.mambarc
 populated config files : C:\Users\charlton\.condarc
       virtual packages : __win=0=0
                          __archspec=1=x86_64
                          __cuda=12.2=0
               channels : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
       base environment : C:\Users\charlton\micromamba
               platform : win-64
```

**Comment by cofinoa** [comment]
It's supposed to be fixed with the merge of PR #8 

Please confirm that it's working.

**Comment by cofinoa** [comment]
Thank you!!!
